LIBSRC/SMB2_SHC/COM/smb2_shc.c

```diff
@@ -370,6 +400,59 @@ int32 __MAPILIB SMB2SHC_GetVoltLevel(enum SHC_PWR_MON_ID pwr_mon_nr, u_int16 *vo
 	return SMB2_SHC_ERR_NO;
 }
 
+/****************************************************************************/
+/** Set power cycle duration in milliseconds
+ *
+ *  \param     status           \IN  desired delay in milliseconds
+ *  \return    SMB2_SHC_ERR_NO on success or error code
+ *
+ *  \sa SMB2SHC_SetPowerCycleDuration
+*/
+int32 __MAPILIB SMB2SHC_SetPowerCycleDuration(u_int16 delay) {
+	u_int8 blkData[SMB_BLOCK_MAX_BYTES];
+		
+        /* set to 1 to enable immediate shutdown */
```

Comment:
What shall be set to 1, blkData[0] or blkData[1]?
Why is blkData[0] always set to 0? 
"    -d=[delay]   Duration of a power cycle (-o flag) in ms \n"
Is it the time between power-down and power-up?
Suggestion: Change "delay" into "duration".